data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -434,15 +432,15 @@
             [/message]
             [message]
                 speaker=Mari
-                message= _ "Nice try, but you used up your turn attacking the quintain. The quintain now gets to attack."
+                message= _ "Nice try, but you already used up your chance to attack. Now it’s the quintain’s turn to strike."
             [/message]
             [message]
                 speaker=Deoran
-                message= _ "The <i>dummy</i> gets a turn?"
+                message= _ "The <i>dummy</i> gets a go?"
             [/message]
             [message]
                 speaker=Mari
-                message= _ "It’s a magical quintain, on loan from the wizards’ academy at Alduin. It strikes twice per attack, and does 13 “impact” damage with every strike. Brace yourself!"
+                message= _ "Why not? It’s a magical quintain, on loan from the wizards’ academy at Alduin. It swings twice per attack, and will sting something fierce if it hits you. Brace yourself!"
```

Comment:
"strikes" seems okay, even in character.

However - it being a turn rather than a strike is significant here, and I'm wondering how it should be reworded. The quintain has already hit back with its counterattacks, the significant detail for this conversation shouldn't be that it can attack, it should be that it gets to decide what to do before Deoran can move away.